.github/PULL_REQUEST_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Pull Request Template
+about: A change is gonna come - Sam Cooke
+title: ''
+labels:
+assignees: ''
+
+---
+
+## What's this PR do?
```

Comment:
What **does** this PR do? 😇

Reply:
Don't know how reliable Merriam-Webster but they say that What's is also a valid contraction of What does.